tools/Makefile

```diff
@@ -109,7 +109,7 @@ verilator-build: ${TBFILES} ${BUILD_DIR}/defines.h test_tb_top.cpp
 	echo '`undef RV_ASSERT_ON' >> ${BUILD_DIR}/common_defines.vh
 	$(VERILATOR)  --cc -CFLAGS ${CFLAGS} $(defines) \
 	  $(includes) -I${RV_ROOT}/testbench -f ${RV_ROOT}/testbench/flist \
-	  -Wno-WIDTH -Wno-UNOPTFLAT ${TBFILES} --top-module tb_top \
+	  -Wno-WIDTH -Wno-UNOPTFLAT -Wno-IMPLICITSTATIC ${TBFILES} --top-module tb_top \
```

Comment:
The `-Wno-IMPLICITSTATIC` option was added in Verilator 5.006. It would be safer to use `-Wfuture-IMPLICITSTATIC -Wno-IMPLICITSTATIC` which tells Verilator <5.006 to ignore the yet unknown option.